pmlc/dialect/pxa/transforms/passes.h

```diff
@@ -18,16 +19,28 @@ class OpPassBase;
 namespace pmlc::dialect::pxa {
 
 struct StencilPass : public mlir::FunctionPass<StencilPass> {
-  StencilPass() = default;
-  StencilPass(const StencilPass &) {}
-  // explicit StencilPass(const StencilPassOptions &options);
+  StencilPass() : costerSet(false) {}
```

Comment:
There is no need for 'costerSet', since std::function has a 'null' version, which is it's default constructor, so you can always just use if (coster) ...